onadata/settings/kc_environ.py

```diff
@@ -153,7 +177,7 @@
 
 # Optional Sentry configuration: if desired, be sure to install Raven and set
 # RAVEN_DSN in the environment
-if 'RAVEN_DSN' in os.environ:
+if os.getenv("RAVEN_DSN") or "" != "":
```

Comment:
I don't understand `"" != ""`

Reply:
@jnm; Would it be better if I had written it like this `if (os.getenv("RAVEN_DSN") or "") != ""`?

Before, when os.getenv("RAVEN_DSN")  was an empty string, the `if` condition was True. 
I wanted to be sure, it wouldn't anymore whether it equals `None` or `""`

Reply:
Ah, interesting. I don't actually know what the operator precedence is here. Does `x or y != z` mean `(x or y) != z` or `x or (y != z)`? Edit: after reading https://docs.python.org/2/reference/expressions.html#operator-precedence, `!=` is "more binding" than `or`, so I think it's the latter.

The code works as-is because it reduces to `if os.getenv("RAVEN_DSN") or False:`, and `bool(None)` and `bool("")` are both `False`.

Reply:
Ok . I'll add the parenthesis.